docs/Hosting on Glitch.md

```diff
@@ -0,0 +1,55 @@
+
+Pros: It works surprisingly well and you shouldn't have any major issues if you follow the instructions in this guide correctly.
+
+Cons: You'll have to wipe old logs every now and then when your upper limit of about 200MB is capped. If you're okay with not having old logs, this is the host for you. Remember, its hard to fill up 200MB worth of text so you should be good for a while.
+
+## Prerequisites
+* Create a Glitch account at https://glitch.com/signin
+> I would recommend registering with your GitHub account. This will come in handy later.
+* Create an UptimeRobot account at https://uptimerobot.com/signUp
+> You will need this to keep your bot up and running 24/7.
+  
+## Bot Setup
+1. Once your account is created, go to this link https://glitch.com/~glitch-discord-modmail and click "View Source".
+2. Then, click the project name in the top left corner and hit "Remix Project".
+3. Now, go to the `.env` file enter your bot token where it says `TOKEN=`. Note there can be no spaces before or after your token in that blank since it is a shell file.
+4. Next, there is a file named `config.json`. There, you will follow the steps from [🛠️ Setting up the bot](docs/setup.md#single-server-setup) and [📝 Configuration](docs/configuration.md) to configure the configuration file to fit your needs.
+
+## Bot Setup - Visual Guide
+<details>
+  <summary>Click to expand!</summary>
+  Step 1: Click Remix Project
+  
+  ![Step1](https://i.imgur.com/y2RA1ve.png)
+  
+  Step 2: Enter your bot token.
+  
+  ![Step2](https://i.imgur.com/lgV1qQS.png)
+  
+  Step 3: Configure config.json to fit your needs. More information in [🛠️ Setting up the bot](docs/setup.md#single-server-setup) and [📝 Configuration](docs/configuration.md).
+  
+  ![Step3](https://i.imgur.com/5RWKcei.png)
+  
+</details>
+
+## UptimeRobot Setup
+![UptimeRobot Logo](https://uptimerobot.com/assets/img/logo_plain.png)
+1. Once your account is created, head over to https://uptimerobot.com/dashboard.php
+2. Click "Add New Monitor"
+3. Select "HTTP(S)"
+4. Set Friendly Name to whatever you want like "ModMail Bot" or something.
+5. For URL, set it to https://your-project-name.glitch.me/ , replacing `your-project-name` with your actual project name. You can also get the link by clicking "Show" in the Glitch project. 
+6. Set Monitoring Interval to every 5 minutes (lowest).
+7. Optionally, you can select an alert contact that it will alert when the project goes offline - I've never used this with any bots.
+8. Click "Create Monitor" (if you don't select an alert contact you might have to click this twice).
+9. You're set with the monitor - it should keep your bot up and running 24/7. Expect occasional downtime, it doesn't happen all the time - just sometimes.
+
+# Additional Resources
+* [Glitch Support Forum](https://support.glitch.com/)
+* [Discord Support Server](https://discord.gg/vRuhG9R) - Though don't bother to ask for Glitch help. They won't be of much help there as they are anti-glitch.
```

Comment:
That is not a glitch support server, it is a support server for the bot.  As the problems arising when trying to use the bot via glitch are not with the bot, but with glitch and the weird things it does and requires, support for glitch or whatever other arbitrary non-standard service you choose is out of scope.  Keep the personal out of your PRs.  There is no reason to mention the bot support server in this guide, as this guide is exclusively for glitch, and the bot support server is referenced in the primary documentation for supported platforms.

Reply:
It's an additional resources tab. It's fine the way it is.

Reply:
We don't want people coming to the server saying the docs said glitch is supported, now teach me. If you want to provide glitch support, do that in DMs.

Reply:
It is not "fine the way it is". This document purports to be a guide for hosting on Glitch.  This mode will not be supported by folks on the Discord server referenced here.  Including that reference is counterproductive because it implies, regardless of the comment following the link, that support will be found there.  Exclude it.